examples/build_twilio_lib.py

```diff
@@ -0,0 +1,70 @@
+import argparse
```

Comment:
What's the plan for the existing script that builds the samples.

Reply:
Once this branch is merged to `main`, we'd need to re-point terraform and go generation from librarian to this build script and deprecate the previous one. Will raise a ticket and link to this. 

Creating a new script so go/terraform releases aren't impacted on merge.

Reply:
Prefer this PR rename the existing file and include the build changes for Java. That gives us better history tracking and diffs on what's changed. Just need to sync the merge with the librarian changes.

Reply:
@childish-sambino Marked changes in same file, will rename the file to `build_twilio_lib.py` before merging.